modules/cf-core/src/node.ts

```diff
@@ -265,7 +265,10 @@ export class Node {
    * @param event
    * @param callback
    */
-  on(event: string, callback: (res: any) => void) {
+  on(
+    event: NodeTypes.EventName | NodeTypes.RpcMethodName,
```

Comment:
👍

Reply:
Also, it's not really making sense that you can subscribe to both `NodeTypes.EventName` and `NodeTypes.RpcMethodName`. Should we consolidate all events you can subscribe to into `NodeTypes.EventName`?